### ISSUE TITLE
 No gradient defined for operation 'SparseSlice'

### ISSUE THREAD

**Issue by lucianspec** [tensorflow/tensorflow#19373]
i am working on solve a indeterminate equation optimization with sparse_slice functions and met this error.
it's strange all sparse functions have gradient except this one. any suggestion how can i avoid this problem?

### Source code / logs
import tensorflow as tf
sparse_values = tf.identity(tf.Variable(tf.constant([1., 2., 3.])))
sparse_indices = tf.constant([[0, 0], [1, 1], [2, 2]], dtype=tf.int64)
sparse_matrix = tf.SparseTensor(sparse_indices, sparse_values, [3, 3])
sparse_slice = tf.sparse_slice(sparse_matrix,[0,0], [2,2])
multiplied = tf.sparse_tensor_dense_matmul(sparse_slice, tf.eye(2))
loss = tf.reduce_sum(multiplied)
gradients = tf.gradients(loss, [sparse_values])
with tf.Session() as session:
    tf.global_variables_initializer().run()
    print(session.run(gradients))

**Comment by sdrelton** [comment]
I'm also running into this issue with a different use case. No workaround I'm afraid but if possible to solve I'd much appreciate it!

**Comment by reedwm** [comment]
/CC @ebrevdo any plans for a sparse slice gradient? Mark as contributions welcome if not.

**Comment by facaiy** [comment]
Hi, I write a gradient op for SparseSlice in #19663. Could you take a look?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ebrevdo: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
